content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -0,0 +1,163 @@
+---
+title: PowerFlex
+linkTitle: "PowerFlex"
+description: >
+  Installing Dell CSI Driver for PowerFlex via Dell CSM Operator
+---
+
+## Installing CSI Driver for PowerFlex via Dell CSM Operator
+
+The CSI Driver for Dell PowerFlex can be installed via the Dell CSM Operator.
+To deploy the Operator, follow the instructions available [here](../../#installation).
+
+Note that the deployment of the driver using the operator does not use any Helm charts and the installation and configuration parameters will be slightly different from the one specified via the Helm installer.
+
+**Note**: MKE (Mirantis Kubernetes Engine) does not support the installation of CSI-PowerFlex via Operator.
+
+### Listing installed drivers with the ContainerStorageModule CRD
+User can query for all Dell CSI drivers using this command:
+`kubectl get csm --all-namespaces`
+
+### Prerequisites
+- If multipath is configured, ensure CSI-PowerFlex volumes are blacklisted by multipathd. See [troubleshooting section](../../../../troubleshooting/powerflex.md) for details
+
+#### SDC Deployment for Operator 
+- This feature deploys the sdc kernel modules on all nodes with the help of an init container.
+- For non-supported versions of the OS also do the manual SDC deployment steps given below. Refer to https://hub.docker.com/r/dellemc/sdc for supported versions.
+- **Note:** When the driver is created, MDM value for initContainers in driver CR is set by the operator from mdm attributes in the driver configuration file, 
+  config.yaml. An example of config.yaml is below in this document. Do not set MDM value for initContainers in the driver CR file manually.
+- **Note:** To use an sdc-binary module from customer ftp site:
+  - Create a secret, sdc-repo-secret.yaml to contain the credentials for the private repo. To generate the base64 encoding of a credential:
+ ```yaml
+      echo -n <credential>| base64 -i
+``` 
+   secret sample to use:
+ ```yaml
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: sdc-repo-creds
+        namespace: vxflexos
+      type: Opaque
+      data:
+        # set username to the base64 encoded username, sdc default is
+          username: <username in base64>
+        # set password to the base64 encoded password, sdc default is
+          password: <password in base64>
+```  
+  - Create secret for FTP side by using the command `kubectl create -f sdc-repo-secret.yaml`.
+  - Optionally, enable sdc monitor by setting the enable flag for the sdc-monitor to true. Please note: 
+    - **If using sidecar**, you will need to edit the value fields under the HOST_PID and MDM fields by filling the empty quotes with host PID and the MDM IPs. 
+    - **If not using sidecar**, please leave this commented out -- otherwise, the empty fields will cause errors.
```

Comment:
We can edit the line to - 
**If not using sidecar**,  No changes are required, since sdc-monitor is disabled by default.